osrm.go

```diff
@@ -100,3 +100,13 @@ func (o OSRM) Match(ctx context.Context, r MatchRequest) (*MatchResponse, error)
 	}
 	return &resp.MatchResponse, nil
 }
+
+// Nearest matches given GPS point to the nearest road network.
+// See https://github.com/Project-OSRM/osrm-backend/blob/master/docs/http.md#service-nearest for details.
```

Comment:
the actual link on the doc is `https://github.com/Project-OSRM/osrm-backend/blob/master/docs/http.md#nearest-service`